### ISSUE TITLE
Ability to view documents attached to Clients

### ISSUE THREAD

**Issue by TekWarren** [invoiceninja/ui#1509]
Currently, you have to "edit" client details to view attached documents as apposed to having say a documents option in this menu:
![image](https://github.com/invoiceninja/ui/assets/866696/b648de35-d3fd-4816-ac15-f3621eec2ee7)

Use case for me is we have clients sign a release of liability waiver and I have begun attaching a digital copy of it to their client record. Going forward we may also like to attach digital copies of things like contracts also. 

Thank you

**Comment by beganovich** [comment]
Hey there,
Thanks for the feature suggestion.

@turbo124 may help us coordinate this as a feature.

**Comment by turbo124** [comment]
@beganovich yes, we can add this as a tab on the clients, i'll provide a route. just to note this will contain all documents that are associated with the user across all entities, so we'll need to follow the design of flutter so that we display the related entity in the table.

cc @Civolilah
